### ISSUE TITLE
How to attach create a sidebar and attach it to a navbar?

### ISSUE THREAD

**Issue by Sanjay-B** [reactstrap/reactstrap#1788]
Hello!

- components: `navbar` & `sidebar`
- reactstrap version `#8.4.1`
- react version `#16.12.0`
- bootstrap version `#4.4.1`

I've been struggling on how to replicate a navbar w/ a fixed sidebar that looks like this:
<img width="1425" alt="Screen Shot 2020-02-13 at 7 37 49 PM" src="https://user-images.githubusercontent.com/22083307/74503145-fab25c80-4ebd-11ea-89f3-132fe1e72474.png">

I've researched for a couple of days and posted a stack overflow question that remains unanswered. Guides relating to this topic is sparse and exist int the form of demos which are hard to follow and poorly organized. https://stackoverflow.com/questions/60218518/whats-the-best-way-to-go-about-creating-a-dashboard-using-react-and-reactstrap

```js
// navbar.js
import { Navbar, Nav, NavbarBrand, NavbarToggler, Collapse, Col, NavItem, NavLink } from "reactstrap";
import React from "react";

export default function Nav_bar({ user }) {
    const style = { marginBottom: "25px" };
    return (
        <div>
            <Navbar color="light" light expand="md" style={style}>
                <NavbarBrand href="/">Web Panel</NavbarBrand>
                <Nav className="ml-auto" navbar>
                    {user && (
                        <>
                            <NavItem>
                                <NavLink href="/discord/logout" className="text-danger">Logout</NavLink>
                            </NavItem>
                        </>
                    )}
                    {(!user) && (
                        <>
                            <NavItem>
                                <NavLink href="/login">Login</NavLink>
                            </NavItem>
                        </>
                    )}
                </Nav>
            </Navbar>
        </div>  
    );
}
```
```js
// _app.js
import React from "react";
import App, { Container as NextContainer } from "next/app";
import Head from 'next/head';
import Nav_bar from '../components/Navbar';
import { Container, Row, Col } from "reactstrap";
import css from '../static/main.css';

class MyApp extends App {
    static async getInitialProps({ Component, ctx }) {
        let pageProps = {};
        if (Component.getInitialProps) {
            pageProps = await Component.getInitialProps(ctx);
        }
            
        if (ctx.req && ctx.req.session.passport) {
            pageProps.user = ctx.req.session.passport.user;
        } 
            
        return { pageProps };
    }

    constructor(props) {
        super(props);
        this.state = {
            user: props.pageProps.user
        };
    }

    render() {
        const { Component, pageProps } = this.props;
        const props = { ...pageProps, user: this.state.user }
        return (
            <div>
                <Head>
                    <title>Core Application</title>
                    <meta name="viewport" content="initial-scale=1.0, width=device-width" />
                    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossOrigin="anonymous"></link>
                </Head>
                {this.state.user && (
                    <>
                        <Nav_bar user={this.state.user} className={ css.Navbar }/>
                        <Container className={ css.Container }>
                            <Row className={ css.Row }>
                                <Component {...props} />
                            </Row>
                        </Container>
                    </>
                )}
                {!this.state.user && (
                    <>
                        <Nav_bar user={this.state.user} />
                        <Container>
                            <Component {...props} />
                        </Container>
                    </>
                )}
            </div>
        );
    }
}
```
``navbar.css``
```css
.ContainerScroller {
    overflow-x: hidden;
    overflow-y: hidden;
}

.FixedNav {
    position: fixed;
    top: 0;
    right: 0;
    left: 0;
    z-index: 1030;
    -webkit-box-orient: horizontal;
    -webkit-box-direction: normal;
    flex-direction: row;
    display: flex;
}

.Navbar {
    width: 70px;
    height: 63px;
    background-color: blue;
}
```
``main.css``
```css
.Container {
    position: absolute;
    margin-top: -1.55rem;
}

.Row {
    position: absolute;
}

.staticDiv {
    margin-top: 1rem;
    margin-left: 1.5rem;
}

.Navbar {
    overflow-y: hidden;
    width: 100%;
}

.MaxedOut {
    width: 100%;
}
```
Result:
<img width="1440" alt="Screen Shot 2020-02-14 at 12 14 32 AM" src="https://user-images.githubusercontent.com/22083307/74503514-1a965000-4ebf-11ea-86a0-0f9270b133a2.png">

Sorry if this is the wrong place to ask. I thought I'd try here. I appreciate any help.

**Comment by kyletsang** [comment]
The sidebar/main content in the bootstrap example is created using Rows and Cols.  If you inspect the source of that page, you can get an idea of which elements to use.

Here's the general gist of it.

```
<Container fluid>
  <Row>
    <Col md={2}>
       Sidebar content
    </Col>
    <Col md={9}>
       Page content
    </Col>
  </Row>
</Container>
```

**Comment by kyletsang** [comment]
Closing as answered
